tab-finder.js

```diff
@@ -67,7 +67,13 @@ window.addEventListener('keydown', (e) => {
  */
 function tabLinkFactory(tab) {
   let tabLink = document.createElement('a');
-  tabLink.textContent = tab.title || tab.id;
+  let title = tab.title || tab.id;
+  if (tab.audible === true) {
+    title += " | 🔊"
+  } else if (tab.mutedInfo.muted === true) {
+    title += " | 🔇"
+  }
```

Comment:
I'm not sure having the audible emoji is as useful as just adding the inaudible one to inaudible tabs.
```suggestion
  if (tab.mutedInfo.muted === true) {
    title += " | 🔇"
  }
```

Reply:
I think it is helpful if you have also an indication of which tab is playing a media stream. Because you may have multiple tabs of the same platform and if they have a playlist feature or something similar the name of the tab also changes but with the audible indication you can still esaly identify that tab.